### DISCUSSION TITLE
Class 'Analytics' not found

### DISCUSSION THREAD

**Discussion by mprince2k18** [freekmurze/laravel-analytics#386] [Q&A]
<?php

namespace App\Http\Controllers;

use Illuminate\Http\Request;
use Illuminate\Support\Collection;
use Analytics;
use LaravelAnalytics;
use Spatie\Analytics\Period;

class AnalyticsController extends Controller
{

    public function index()
    {
        return \Analytics::fetchMostVisitedPages(Period::days(7));
    }

    //END
}

This is the controller code. I am facing Class 'Analytics' not found.

**Comment by igcse-library** [comment]
hi i have this same error .. how can we resove
